std/range/package.d

```diff
@@ -7723,25 +7723,25 @@ if (isForwardRange!Source && hasLength!Source)
 A fixed-sized sliding window iteration
 of size `windowSize` over a `source` range by a custom `stepSize`.
 
-The `Source` range must be at least an `ForwardRange` and
-the `windowSize` must be greater than zero.
+The `Source` range must be at least an $(REF_ALTTEXT ForwardRange, isForwardRange, std,range,primitives)
+and the `windowSize` must be greater than zero.
 
 For `windowSize = 1` it splits the range into single element groups (aka `unflatten`)
 For `windowSize = 2` it is similar to `zip(source, source.save.dropOne)`.
 
 Params:
-    f = Whether the last element with fewer elements than `windowSize`
-        should be be ignored (`Yes.withPartial`)
+    f = Whether if the last element has fewer elements than `windowSize`
```

Comment:
Just saying `Whether the` is more colloquially correct, if not grammatically correct.